### PR TITLE
doc:  add kafka plugin source readme

### PR DESCRIPTION
**Pull request by huolibo**

### CODE REVIEW

File: doc/source.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+   poll.interval.ms=1000
+   topic.prefix=tdengine-
+   fetch.max.rows=100
+   out.format=line
+   key.converter=org.apache.kafka.connect.storage.StringConverter
+   value.converter=org.apache.kafka.connect.storage.StringConverter
+
+   ```
+
+   The `connection.url`,`connection.database` specify the connection URL,database name of the TDengine server. By
```
Comment (dingbo8128) [inline doc/source.md:43]: need space between "," and following content.

File: doc/source.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+
+- Type:string
+- Importance:high
+- Default: ""
+
+### timestamp.initial
+
+The timestamp used for initial queries. If not specified, all data will be retrieved. format is `yyyy-MM-dd HH:mm:ss` 
+
+- Type:string
```
Comment (dingbo8128) [inline doc/source.md:195]: need space after ":"
you may need a markdown format tool.

**Review by dingbo8128 — APPROVED** [review]

File: doc/source.md (hunk 2 above)
Comment (huolibo) [reply doc/source.md]: do as you say

File: doc/source.md (hunk 1 above)
Comment (huolibo) [reply doc/source.md]: do as you say

**Review by sangshuduo — APPROVED** [review]
LGTM

**Pull request merged**